Resources/doc/installation.rst

```diff
@@ -31,38 +31,36 @@ To install DoctrineMongoDBBundle with Composer run the following command:
 
     composer require doctrine/mongodb-odm-bundle
 
-All that is left to do is to update your ``AppKernel.php`` file, and
-register the new bundle:
+
+Enable the Bundle
```

Comment:
Original intention was that this text belongs to `Install the bundle with Composer` chapter as the bundle will be automatically enabled when using Flex.

Reply:
Can this section be dropped entirely then? I left it only for completeness and added sentence `Your bundle should be automatically enabled if you use Flex`, but I agree it doesn't provide much value.

Reply:
I'm not entirely sure what I had in mind 2 weeks ago, on a fresh look this looks OK. So let's ignore my original comment :)